.travis.yml

```diff
@@ -2,6 +2,9 @@ language: java
 
 sudo: false
 
+before_install:
+  - export M2_HOME=/usr/local/maven-3.3.9
+
```

Comment:
It is because the maven invoker use environment variable `M2_HOME` or system property `maven.home` to get the maven binary. Without this, we would have an exception like this: https://travis-ci.org/Luolc/regression-tool/jobs/258651414#L1528

FYI: https://maven.apache.org/shared/maven-invoker/apidocs/org/apache/maven/shared/invoker/Invoker.html#setMavenHome(java.io.File),
http://svn.apache.org/viewvc/maven/shared/tags/maven-invoker-3.0.0/src/main/java/org/apache/maven/shared/invoker/MavenCommandLineBuilder.java?view=markup#l113